app/views/application/about.html.erb

```diff
@@ -140,22 +157,20 @@
 <p>We check the following things about the CSV that you publish:</p>
 
 <ul>
-<li>that there are no encoding errors because of odd characters in the file</li>
-<li>all the line breaks should be the same</li>
-<li>there should be a header line</li>
-<li>all the columns should have a name</li>
-<li>all the column names should be different</li>
-<li>every row in the file should have the same number of columns</li>
-<li>none of the rows should be blank</li>
-<li>all the quotes should be matched up</li>
-<li>there shouldn't be any whitespace between commas and double quotes around fields</li>
+<li><b>Invalid encoding: </b>that there are no encoding errors because of odd characters in the file</li>
+<li><b>Line breaks: </b>all the line breaks should be the same</li>
+<li><b>Undeclared header: </b>there should be a header line</li>
+<li><b>Ragged rows:</b> every row in the file should have the same number of columns</li>
+<li><b>Blank rows:</b> none of the rows should be blank</li>
+<li><b>Stray/Unclosed quote:</b> all the quotes should be matched up</li>
+<li><b>Whitespace:</b> there shouldn't be any whitespace between commas and double quotes around fields</li>
 </ul>
```

Comment:
This flows a bit strangely - it sounds like we check for invalid encoding. I'd say something like this:

```
<li><b>Encoding: </b>that there are no encoding errors because of odd characters in the file</li>
<li><b>Line breaks: </b>all the line breaks should be the same</li>
<li><b>Headers: </b>there should be a header line</li>
<li><b>Number of columns:</b> every row in the file should have the same number of columns</li>
<li><b>Blank rows:</b> none of the rows should be blank</li>
<li><b>Quotes:</b> all the quotes should be matched up</li>
<li><b>Whitespace:</b> there shouldn't be any whitespace between commas and double quotes around fields</li>
```